### PR TITLE
Fix public loading

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- Previously a solution was made to fix the about files not being found in dev
- Now the solution is more broad spectrumed and syns all public files as a similar issue was encountered with branding

### CODE REVIEW

**Review by gsfk — CHANGES_REQUESTED** [review]
Not working for me: 

```
bentov2-public  |  /bento-public/bento-public/src/public/assets/bento.svg doesn't exist
```
Looks like it's wiping out stuff that's expected to be there.

File: docker-compose.local.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ services:
       - ./packs:/packs
       - ${PWD}/lib/public/en_about.html:/bento-public/src/public/en_about.html
       - ${PWD}/lib/public/fr_about.html:/bento-public/src/public/fr_about.html
+      - ${PWD}/lib/public/branding.png:/bento-public/src/public/branding.png
```
Comment (gsfk) [inline docker-compose.local.yaml:21]: corrrect path appears to be `/bento-public/src/public/assets/branding.png`, missing `/assets/`

**Review by gsfk — CHANGES_REQUESTED** [review]
Needs a fix for container path for branding, haven't tried translations yet

File: docker-compose.local.yaml (hunk 1 above)
Comment (SanjeevLakhwani) [reply docker-compose.local.yaml]: Yes, correct it

**Review by gsfk — APPROVED** [review]

**Pull request merged**